atomate/vasp/fireworks/core.py

```diff
@@ -284,6 +324,89 @@ def __init__(self, structure, name="static dielectric", vasp_cmd="vasp",
             structure.composition.reduced_formula, name), **kwargs)
 
 
+class LcalcpolFW(Firework):
```

Comment:
Let's talk sometime about this FW vs the StaticInterpolate

Reply:
I don't seem to see StaticInterpolateFW used in the main workflow so I am a little confused